File: src/targets/node/fetch/fixtures/multipart-file.js
import fs from 'fs';

const formData = new FormData();
formData.append('foo', await fs.openAsBlob('src/fixtures/files/hello.txt'));

const url = 'https://httpbin.org/anything';
const options = {method: 'POST', body: formData};

fetch(url, options)
  .then(res => res.json())
  .then(json => console.log(json))
  .catch(err => console.error(err));

File: src/targets/csharp/restsharp/fixtures/multipart-file.cs
var options = new RestClientOptions("https://httpbin.org/anything");
var client = new RestClient(options);
var request = new RestRequest("");
request.AlwaysMultipartFormData = true;
request.FormBoundary = "---011000010111000001101001";
request.AddFile("foo", "src/fixtures/files/hello.txt");
var response = await client.PostAsync(request); 